### ISSUE TITLE
Issue starting server

### ISSUE THREAD

**Issue by littlejohnjt** [jmcooper/ng2-fundamentals#16]
I have installed the necessary tools and executed npm install with no errors.  When attempting to statt the server using npm install as instructed I get the following error show in the attached log file.

[npm-debug.zip](https://github.com/jmcooper/ng2-fundamentals/files/1049151/npm-debug.zip)

**Comment by joeeames** [comment]
would you mind posting this in the course forums?

**Comment by littlejohnjt** [comment]
Done!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by joeeames** [comment]
I can't see it on the forums, although I got the notification of the comment. You didn't delete it did you?

**Comment by littlejohnjt** [comment]
No, I did not.  I am out for a run, ill check it when I get back.

Sent from my Windows 10 phone

**Comment by littlejohnjt** [comment]
I really like the course so far, good teaching style!!

Sent from my Windows 10 phone

**Comment by littlejohnjt** [comment]
Joe,

For some reason it is not in the discussion forum.  Let me try again.


  *   Todd

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by littlejohnjt** [comment]
I have posted it again.  Let me know if you have any questions.  Thanks again!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by littlejohnjt** [comment]
It was tagging it as spam.  I have went in and told DISQUS that it is not spam.  Hopefully you'll see it within the next day or so.


- Todd


J. Todd Littlejohn - 13110 Marsh Fern Dr. - Orlando, FL  32828 - C: 202.664.7798

________________________________
